patches/minecraft/net/minecraft/world/item/enchantment/DiggingEnchantment.java.patch

```diff
@@ -0,0 +1,10 @@
+--- a/net/minecraft/world/item/enchantment/DiggingEnchantment.java
++++ b/net/minecraft/world/item/enchantment/DiggingEnchantment.java
+@@ -22,6 +_,6 @@
+    }
+ 
+    public boolean m_6081_(ItemStack p_44668_) {
+-      return p_44668_.m_150930_(Items.f_42574_) ? true : super.m_6081_(p_44668_);
++      return p_44668_.m_41720_() instanceof net.minecraft.world.item.ShearsItem ? true : super.m_6081_(p_44668_);
```

Comment:
Is this intentionally still `instanceof`? I suppose a non-shears class could always override the can be enchanted method

Reply:
This change is carried over from the original PR 7544. But actually you are right that the "can be enchanted" method could be overridden by every modded shears item. I can reverse this change here.

Reply:
I assume most modders would expect making their own instance of the shears class would be enchantable automatically. Might be nice if having the tool action allowed it as well. That said, we could save a patch by doing neither, so its not really my call.

Reply:
I also think modders would expect that. So I left the patch in there. A ToolAction would be kind of strange in this case, since it is not really an action. It would be called "shears_digging_echantable" (or similar) and it would not be good for anything other than this case.